assets/js/components/Beneficiaries.vue

```diff
@@ -271,17 +328,37 @@ export default Vue.extend({
       /*
        *
        */
-      bentered.append("title").text(
+      bentered.attr("title",
```

Comment:
is this how the author suggests you use the library, in his extremely simple, extremely obvious, extremely direct example?